content/donate.md

```diff
@@ -0,0 +1,20 @@
+---
+permalink: /donate/
+layout: donate
+title: Solar Chapter
+custom_css:
+  - '/assets/css/donate.css'
+# Content
+background_image: |
+  https://res.cloudinary.com/db1a3rvdf/image/upload/v1585437223/Solar%20Chapter%20Website/donation_page/donate_qlhpwz.jpg
```

Comment:
Can you change this picture to not be kain makna specific? Anything that has more tone towards donation?

Reply:
Address separately